presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -0,0 +1,254 @@
+==========================================
+Presto Coordinator Kerberos Authentication
+==========================================
+
+The Presto server can be configured to enable Kerberos authentication over
+HTTPS for clients such as the :doc:`Presto CLI </security/cli>` or the various
+JDBC and ODBC drivers.
+
+.. warning::
+
+   Unauthenticated access over HTTP *must* be enabled because worker nodes
+   cannot yet be configured to connect to the coordinator using HTTPS or to
+   authenticate with Kerberos. It is the adminstrator's responsibility to
+   ensure that insecure access to the Presto coordinator is blocked for any
+   node that isn't a worker node.  This can be done by blocking access to the
+   Presto coordinator's HTTP port from nodes that are not workers.
+
+All of the Presto configuration changes are made on the coordinator. There are
+no changes required to the worker configuration to enable Kerberos
+authentication.
+
+Configuring the Environment
+---------------------------
+
+.. |subject_node| replace:: client
+.. include:: kerberos-services.fragment
+.. include:: kerberos-configuration.fragment
+
+Kerberos Principals and Keytab Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Presto server needs a Kerberos principal, as do users who are going to
+connect to the Presto server. You will need to create these users in Kerberos
+using `kadmin <http://web.mit.edu/kerberos/krb5-latest/doc/admin/admin_commands/kadmin_local.html>`_.
+
+Additionally, the Presto server needs a `keytab file
+<http://web.mit.edu/kerberos/krb5-devel/doc/basic/keytab_def.html>`_. The
+keytab file can be created using :command:`kadmin` after you create the
+principal.
```

Comment:
Suggestion:

After you create the principal, you can create the keytab file using....